core/src/main/res/layout/dialog_add_note.xml

```diff
@@ -5,6 +5,14 @@
   android:layout_height="match_parent"
   android:background="@android:color/transparent">
 
+  <org.kiwix.kiwixmobile.core.utils.NestedCoordinatorLayout
```

Comment:
What's the use of this view here it looks unused, without this snackbar is working fine.

Reply:
Yes you are right. It is redundant. I'll be removing NestedCoordinatorLayout then.